spec/lib/aoc_rb/app_spec.rb

```diff
@@ -435,4 +435,22 @@
       expect(File.read(part_2_file)).to eq markdown_part_2
     end
   end
+
+  describe "output" do
+    before do
+      stub_request(:get, "https://adventofcode.com/2018/day/4/input").to_return({body: 'test'})
+    end
+
+    it "can return output" do
+      year = 2018
+      day = 4
+      AocRb::App.start %w(bootstrap -y 2018 -d 4)
+
+      challenge_dir = File.join(File.dirname(__FILE__), "../../..", "challenges", "#{year}", "#{day.to_s.rjust(2, "0")}")
+      solution_for_day = File.join(challenge_dir, "solution.rb")
+      require solution_for_day
```

Comment:
We may as well simplify this:
```suggestion
      AocRb::App.start %w(bootstrap -y 2018 -d 4)
      require File.join(File.join(File.dirname(__FILE__), "../../..", "challenges", "2018", "04"), "solution.rb")
```